### ISSUE TITLE
waydroid no module named gbinder (gbinder is installed)

### ISSUE THREAD

**Issue by No0ne003** [waydroid/waydroid#1386]
### Describe the bug

whatever i write any command with Waydroid this error show
```sh
Traceback (most recent call last):
  File "/usr/bin/waydroid", line 7, in <module>
    import tools
  File "/usr/lib/waydroid/tools/__init__.py", line 12, in <module>
    from . import actions
  File "/usr/lib/waydroid/tools/actions/__init__.py", line 3, in <module>
    from tools.actions.initializer import init, wait_for_init, remote_init_client
  File "/usr/lib/waydroid/tools/actions/initializer.py", line 5, in <module>
    from tools import helpers
  File "/usr/lib/waydroid/tools/helpers/__init__.py", line 5, in <module>
    import tools.helpers.props
  File "/usr/lib/waydroid/tools/helpers/props.py", line 8, in <module>
    from tools.interfaces import IPlatform
  File "/usr/lib/waydroid/tools/interfaces/IPlatform.py", line 1, in <module>
    import gbinder
ModuleNotFoundError: No module named 'gbinder'
```
I already installed gbinder with 
```sh
paru -S python-gbinder
```
and it installed with no error

the problem start just today (after updating) it was work fine for a long time

### Waydroid version

1.4.2

### Device

Linux Desktop

### Operating System

arch linux

### Kernel version

6.8.7-arch1-2

### Desktop Environment

i use wayland (hyrpland)

### GPU

intel HD Graphics 520 GPU

### Logs

Traceback (most recent call last):
  File "/usr/bin/waydroid", line 7, in <module>
    import tools
  File "/usr/lib/waydroid/tools/__init__.py", line 12, in <module>
    from . import actions
  File "/usr/lib/waydroid/tools/actions/__init__.py", line 3, in <module>
    from tools.actions.initializer import init, wait_for_init, remote_init_client
  File "/usr/lib/waydroid/tools/actions/initializer.py", line 5, in <module>
    from tools import helpers
  File "/usr/lib/waydroid/tools/helpers/__init__.py", line 5, in <module>
    import tools.helpers.props
  File "/usr/lib/waydroid/tools/helpers/props.py", line 8, in <module>
    from tools.interfaces import IPlatform
  File "/usr/lib/waydroid/tools/interfaces/IPlatform.py", line 1, in <module>
    import gbinder
ModuleNotFoundError: No module named 'gbinder'

**Comment by forlornlamp** [comment]
Can confirm, I get the same errors in my journal.  `pacman -Ql python-gbinder` returns:
```
python-gbinder /usr/
python-gbinder /usr/lib/
python-gbinder /usr/lib/python3.11/
python-gbinder /usr/lib/python3.11/site-packages/
python-gbinder /usr/lib/python3.11/site-packages/gbinder.cpython-311-x86_64-linux-gnu.so
python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/
python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/PKG-INFO
python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/SOURCES.txt
python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/dependency_links.txt
python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/top_level.txt
```

**Comment by No0ne003** [comment]
> Can confirm, I get the same errors in my journal. `pacman -Ql python-gbinder` returns:
> 
> ```
> python-gbinder /usr/
> python-gbinder /usr/lib/
> python-gbinder /usr/lib/python3.11/
> python-gbinder /usr/lib/python3.11/site-packages/
> python-gbinder /usr/lib/python3.11/site-packages/gbinder.cpython-311-x86_64-linux-gnu.so
> python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/
> python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/PKG-INFO
> python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/SOURCES.txt
> python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/dependency_links.txt
> python-gbinder /usr/lib/python3.11/site-packages/gbinder_python-1.1.2-py3.11.egg-info/top_level.txt
> ```

I fix it by this command ```paru -S python-gbinder --rebuil```

**Comment by laichiaheng** [comment]
Weird, why should I rebuild it if there is no update?

**Comment by cniw** [comment]
> Weird, why should I rebuild it if there is no update?

Because there is an update on python side from 3.11.x to 3.12.x read the [commit here](https://gitlab.archlinux.org/archlinux/packaging/packages/python/-/commit/f8d696dbe2c7275f67113c20f8001d29f1f36950), so all package that has python modules need to be rebuild

**Comment by ipaqmaster** [comment]
This was driving me mad because you're right but my rebuilds aren't doing the trick.

Turns out the (Archlinux) jenkins server responsible for building my packages (including `python-gbinder`) was 'fully up to date' but upon rebuilding this package the problem didn't resolve.

Turns out the package mirrors it was using hadn't synced with upstream for a while. I ran `sudo reflector -c AU --ipv4 -p https,rsync --save /etc/pacman.d/mirrorlist` to get a new list of mirrors and it has updated a few hundred packages. 

Naturally rebuilding solved this problem this time around as expected. Never using those responsible `pacman` mirrors again 🤷
